src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,27 +205,38 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    driverXbox.a().onTrue(elevatorArm.setSetpointCommand(Setpoint.kL3));
-    driverXbox.x().onTrue(elevatorArm.setSetpointCommand(Setpoint.kIdleSetpoint));
-    driverXbox.b().whileTrue(intake.intakeGamepiece());
-    driverXbox.y().onTrue(Commands.none());
-    driverXbox.start().onTrue(Commands.runOnce(drivebase::zeroGyro));
-    driverXbox.back().onTrue(Commands.none());
-    driverXbox.leftTrigger().onTrue(Commands.none());
-    driverXbox.rightTrigger().onTrue(endEffector.outtake());
+    driverXbox.a().onTrue(elevatorArm.setSetpoint(Setpoint.kL3));
     driverXbox
-        .leftBumper()
+        .x()
         .whileTrue(
             Commands.deferredProxy(() -> drivebase.driveReef(FieldConstants.Direction.LEFT)));
     driverXbox
-        .rightBumper()
+        .b()
         .whileTrue(
             Commands.deferredProxy(() -> drivebase.driveReef(FieldConstants.Direction.RIGHT)));
+    driverXbox.y().onTrue(elevatorArm.setSetpoint(Setpoint.kIdleSetpoint));
+    driverXbox.start().onTrue(Commands.runOnce(drivebase::zeroGyro));
+    driverXbox.back().onTrue(Commands.none());
+    driverXbox.leftBumper().whileTrue(climber.reverseClimb());
+    driverXbox.rightBumper().whileTrue(climber.climb());
+    driverXbox.leftTrigger().whileTrue(pivotAndIntake());
+    driverXbox.rightTrigger().onTrue(outtakeCoral);
 
-    codriverXbox.a().onTrue(Commands.none());
-    codriverXbox.x().onTrue(Commands.none());
-    codriverXbox.b().onTrue(Commands.none());
-    codriverXbox.y().onTrue(Commands.none());
+    codriverXbox.a().onTrue(elevatorArm.setSetpoint(Setpoint.kL1));
+    codriverXbox.x().onTrue(elevatorArm.setSetpoint(Setpoint.kL2));
+    codriverXbox
+        .b()
+        .onTrue(
+            Commands.sequence(
+                new ParallelCommandGroup(
+                        elevatorArm.setSetpoint(Setpoint.kIntake), endEffector.intake())
+                    .until(elevatorArm::atSetpoint),
+                endEffector.stall(),
+                new ParallelCommandGroup(elevatorArm.setSetpoint(Setpoint.kHover))
+                    .until(elevatorArm::atSetpoint),
```

Comment:
```suggestion
                elevatorArm.setSetpoint(Setpoint.kHover).until(elevatorArm::atSetpoint),
```
Don't need parallel here since it's only one command.